pkg/edgeview/src/policy.go

```diff
@@ -238,6 +247,27 @@ func checkIPportPolicy(tcpOpt string, evStatus *types.EdgeviewStatus) (bool, str
 	return true, appName, ""
 }
 
+// checkAddrKube - check if the IP address is in the kube network
+// those are the k3s specific default network prefixes
+func checkAddrKube(addr string) bool {
+	ipa := net.ParseIP(addr)
+	if ipa == nil {
+		return false
+	}
+
+	_, subnet1, err := net.ParseCIDR("10.42.0.0/16")
```

Comment:
Are these fixed address ranges (same for line 263)? Should these be hard-coded in here?

Reply:
those are the default of k3s setup. we don't change that. see the link: https://docs.k3s.io/cli/server in the 'Networking' section of defaults. but for 10.42.0.0/16, I can actually modify this to check the interface prefix of 'cni0' and get from there. for '10.43.0.0/16', we just have to define it here, there is nothing i can gather from the system easily.